### ISSUE TITLE
[TF1] ERROR: no suitable InfoExtractor for URL

### ISSUE THREAD

**Issue by sm1090** [rg3/youtube-dl#8599]
Hi !

Thanks a lot for your great software !

Right now I encounter the following error when trying to download from tf1.fr. For instance :

`youtube-dl` --verbose `http://www.tf1.fr/tf1/le-secret-d-elise/videos/secret-d-elise-episode-4.html`

[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--verbose', u'http://www.tf1.fr/tf1/le-secret-d-elise/videos/secret-d-elise-episode-4.html']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.02.13
[debug] Python version 2.7.11 - Linux-4.3.0-1-amd64-x86_64-with-debian-stretch-sid
[debug] exe versions: ffmpeg 2.8.5-1, ffprobe 2.8.5-1
[debug] Proxy map: {}
[TF1] secret-d-elise-episode-4: Downloading webpage
[TF1] secret-d-elise-episode-4: Downloading JSON metadata
ERROR: no suitable InfoExtractor for URL 
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/bin/youtube-dl/__main__.py", line 19, in <module>
    youtube_dl.main()
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 411, in main
    _real_main(argv)
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 401, in _real_main
    retcode = ydl.download(all_urls)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1692, in download
    url, force_generic_extractor=self.params.get('force_generic_extractor', False))
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 677, in extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 729, in process_ie_result
    extra_info=extra_info)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 692, in extract_info
    self.report_error('no suitable InfoExtractor for URL %s' % url)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 541, in report_error
    self.trouble(error_message, tb)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 503, in trouble
    tb_data = traceback.format_list(traceback.extract_stack())

The same error occurs when trying several other url from tf1.fr (I actually found no working url), however it used to work on week ago. Things may have changed on the TF1 side.

Cheers !

**Comment by dstftw** [comment]
This video is geo restricted. Can you watch it in browser?

**Comment by Crypt0Graf** [comment]
The problem with TF1 is that they discontinued [wat.tv](http://www.wat.tv/) , from which TF1 embedded the Videos. If you're trying to download a video from TF1,  you'll get "ERROR: Unable to extract wat id; please report this issue on https://yt-dl.org/bug".
Usually, downloading videos from wat.tv was possible even for geo-restricted videos and even when you were unable to watch them in your browser.

In other words: TF1 is broken and need to be reimplemented in youtube-dl.

**Comment by sm1090** [comment]
Well, I use a VPN to unlock the geo-restriction, and I indeed can watch the video in the browser.

**Comment by Animis09** [comment]
Same problem from here. 
**ERROR: no suitable InfoExtractor for URL**

I'm not georestricted.

**Comment by remitamine** [comment]
will be fixed in the next version.
